addons/im_livechat/models/discuss_channel.py

```diff
@@ -441,8 +441,19 @@ def _email_livechat_transcript(self, email):
             'email_from': company.catchall_formatted or company.email_formatted,
             'author_id': self.env.user.partner_id.id,
             'email_to': email_split(email)[0],
+            'body': mail_body,
             'body_html': mail_body,
         })
+        mail.body_html = self.env['mail.render.mixin']._render_encapsulate(
```

Comment:
Would prepare body_html and add it directly when creating the mail, to avoid a write just after the create.